cdc/pkg/regionspan/span.go

```diff
@@ -37,6 +38,15 @@ func (s Span) String() string {
 	return fmt.Sprintf("[%s, %s)", hex.EncodeToString(s.Start), hex.EncodeToString(s.End))
 }
 
+func (s Span) ID() uint64 {
```

Comment:
Yes, it is indeed possible. But I don't think adding a seed is effective, because this is a probabilistic event.

Reply:
a simple solution, https://github.com/zeminzhou/migration/blob/tikv-cdc/cdc/cdc/owner/scheduler_v1.go#L446-L453.

Reply:
There is still conflict between "currentKeySpans" and the keyspans acquired before.

Reply:
How about using region id as keyspan id?https://github.com/zeminzhou/migration/blob/tikv-cdc/cdc/cdc/owner/scheduler_v1.go#L444.

If we do it this way, keyspan will lose its meaning.

Reply:
No. The range of a region will change after split or merge.

How about add a `seed` to `KeySpan` to avoid conflict ?

We maintain all key spans in owner. Every time we get a new key span, calculate the hash with `seed` starting from 0. If the hash conflicts with an existing key span, `seed += 1`, and calculate the hash again. Until we find a hash without conflict.
Note that the `seed` should be transfer to Capture & Processor, along with key span's start & end.